src/components/SelectMateria.js

```diff
@@ -72,14 +86,79 @@ const SelectMateria = ({ materiasToShow }) => {
           placeholder="Buscar Materia..."
           _placeholder={{ color: "gray.200" }}
         />
-        <InputRightElement children={<SearchIcon color="primary.500" />} />
+        <InputRightElement
+          children={
+            <Menu closeOnSelect={false}>
+              <MenuButton
+                colorScheme="primary"
+                variant="outline"
+                border="none"
+                as={Button}
+                rightIcon={
+                  <Tooltip placement="top" label="Filtrar materias">
+                    <Icon boxSize={5} mr={2} viewBox="0 0 24 24">
+                      <svg
+                        height="200px"
+                        width="200px"
+                        stroke="currentColor"
+                        fill="currentColor"
+                        strokeWidth="0"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
```

Comment:
```suggestion
                      <svg
                        fill="currentColor"
                      >
```

me parece que estas propiedades sobran, creo que `boxSize` ya hace todo lo de tamaño solo